apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -113,18 +117,42 @@ export function Chat({
-          threadId:
-            params.threadId || isNewChat ? threadId : activeThread?.threadId,
-          content: message.content,
-          jwt: session!.user?.hasuraJwt,
-        });
+      async onFinish(message) {
+        await Promise.all([
+          saveNewMessage({
+            role: "user",
+            threadId: params.threadId || isNewChat ? threadId : activeThread?.threadId,
+            content: userContentRef.current,
+            jwt: session!.user?.hasuraJwt,
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)